### PR TITLE
Switch to Trusty on travis

### PR DESCRIPTION
**Pull request by shioyama**
Builds are failing on postgres/rbx combination only. Proposed solution is to use latest version of rubinius:

Ref: https://twitter.com/shioyama/status/779813304713646081

### CODE REVIEW

**Comment by shioyama** [comment]
This is a positively painful experience. If anybody with any idea how to get rubinius builds to work on Travis CI, please let me know.

**Comment by shioyama** [comment]
@brixen Trying to get Rubinius to work again, updated to use Trusty and fixed issues with other builds but rbx 3.60 is still failing... will keep at it, let me know if you can provide any hints :smile:

**Comment by shioyama** [comment]
Correction, the mysql build seems to be working, but sqlite and postgres are failing on `bundle install`...

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,9 @@ language: ruby
 cache: bundler
 sudo: required
 dist: trusty
-before_install: gem install bundler
+before_install:
+  - gem install bundler
+  - mysql -u root -e 'create database globalize_test character set utf8;' >/dev/null
```
Comment (dalpo) [inline .travis.yml]: What do think about to put the  `character set utf8` argument directly here? 
https://github.com/globalize/globalize/blob/master/test/support/database.rb#L58

Comment (shioyama) [inline .travis.yml]: Sure makes sense. I just want to figure out why rubinius builds are failing...

**Comment by shioyama** [comment]
Ok I give up. I can't get rubinius to work and with the update to Trusty jruby doesn't seem to `bundle` either...

**Comment by simi** [comment]
@shioyama don't worry, I try to take a look also.

**Comment by brixen** [comment]
@shioyama I opened https://github.com/globalize/globalize/pull/554 to work on Rubinius compatibility, not to replace this PR but so I can separate updates from other changes you may need to make for other engines (eg why does JRuby not bundle on Trusty? Is that a blocker to using Trusty?)

For this PR, could you revert the change to using `rbx-3.60`? Just using `rbx` should work and using a more specific value will inhibit using newer releases as I fix issues.

**Comment by shioyama** [comment]
@brixen Thanks for all your investigating. I removed the commit switching to rbx-3.60.

About JRuby, yes I believe that would be a blocker for switching to Trusty.

I'll look into #554 later today.

**Comment by brixen** [comment]
@shioyama quick update, I was able to get building Rubinius Precise binaries working again https://travis-ci.org/rubinius/rubinius-build/builds/162983586

**Comment by shioyama** [comment]
@brixen Brilliant, thanks!

**Comment by brixen** [comment]
@shioyama omg, sorry for the tease. Unfortunately, using the LLVM-built packages requires using an alternate build of GCC and libc++, which isn't installed by default on Precise, so the binary tries to load a shared lib version of libatomic, which fails. :( I'm trying to figure out if I can statically link.

**Comment by brixen** [comment]
@shioyama ok, bad news. I've now spent a few hours trying to get binaries to build for Precise and statically link the necessary libraries. I have not been successful and I'm not going to spend any more time. I'd suggest moving all Rubinius to `allow_failures` for Precise.

It's a tragedy of our collective infrastructure that Precise is still in use at all. If I understand Ubuntu LTS, it's only got a little over 6 months remaining. Orgs like Heroku have long since moved forward. Whatever the issue is with JRuby and Bundler on Trusty, it will need to be resolved sooner than later completely independent of anything Rubinius related.

I'll leave #554 open and continue to work on getting Rubinius green on Trusty. If this doesn't work for you, I understand.

Packaging and distribution has been an utter nightmare for Rubinius given the intricate, obscure, and painful interdependencies that exist in our OS infrastructure, like we see here. I've recently removed support for 32-bit, GCC, and won't support platforms for which no LLVM packages exist. This obviously limits the platforms that we support, but the alternative (supporting all kinds of legacy systems with multi-year out-of-date packages) becomes more untenable by the day.

Thanks for understanding and sorry for the frustrating experience.

**Comment by brixen** [comment]
@shioyama and RVM is again failing to find the Rubinius 14.04 binaries despite that they are where they they have always been (see links in https://twitter.com/brixen/status/780979102857572352).

**Comment by shioyama** [comment]
@brixen No apologies needed! Thank *you* for taking so much time to address this issue, to the point of actually looking at the gem dependencies. I think you're absolutely right that we'll have to deal with the transition to Trusty eventually so now's as good a time as any. I can take it from here, thanks again!

**Comment by parndt** [comment]
@shioyama should we just move this to allowed failures?

**Comment by shioyama** [comment]
@parndt rbx is already in [allowed failures](https://github.com/globalize/globalize/blob/master/.travis.yml#L28-L33). You mean switch to trusty with rbx on allowed failures? I guess since we're already allowing them to fail it wouldn't change much.

**Comment by parndt** [comment]
@shioyama I mean that there are three rbx failures on your PR:
https://travis-ci.org/globalize/globalize/jobs/163254274
https://travis-ci.org/globalize/globalize/jobs/163254275
https://travis-ci.org/globalize/globalize/jobs/163254276

Those are the ones I'm asking about 😄

**Pull request closed without merging**